packages/babel-register/test/index.js

```diff
@@ -156,20 +156,18 @@ describe("@babel/register", function () {
     // requires() another with @babel/register active, and I couldn’t get
     // that working inside a test, possibly because of jest’s mocking
     // hooks, so we spawn a separate process.
-
-    spawnNode(["-r", registerFile, sourceMapTestFile], output => {
-      let err;
-
-      try {
-        const sourceMap = JSON.parse(output);
-        expect(sourceMap.map.sourceRoot + sourceMap.map.sources[0]).toBe(
-          sourceMapNestedTestFile,
-        );
-      } catch (e) {
-        err = e;
-      }
-
-      callback(err);
+    return new Promise((resolve, reject) => {
+      spawnNode(["-r", registerFile, sourceMapTestFile], output => {
```

Comment:
We can make `spawnNode` return a promise, and just use an async function for tests here.